setup.py

```diff
@@ -29,15 +30,17 @@
     "hydra-core>=1.3.2",
     "iopath>=0.1.10",
     "pillow>=9.4.0",
+    "scipy>=1.14.0",
 ]
 
 EXTRA_PACKAGES = {
     "demo": ["matplotlib>=3.9.1", "jupyter>=1.0.0", "opencv-python>=4.7.0"],
```

Comment:
When trying to work off this branch, I ran into an issue with `matplotlib>=3.9.1`. Doing `matplotlib>=3.9.0` helped.

Separately, I had to comment out `ext_modules` in the main `setup()` function.

Reply:
I ran into the same problem with `matplotlib>=3.9.1` and `>=3.9.0` fixed it. Thanks!
Didn’t have any issues with `ext_modules` though.